labextension/.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
```

Comment:
What's the purpose of this new file?

Reply:
This is the yarn 3 configuration file. If you develop the extension against jupyterlab 4, `jlpm` is actually yarn 3 instead of yarn 1 in the case of lab 3

Reply:
Thanks! Does every extension repo need this config file, or is it a property of the development environment? Does the package registry have to be hardcoded, or can we rely on the developers default e.g. if they're using an internal mirror?

Reply:
> Thanks! Does every extension repo need this config file

Yes

> Does the package registry have to be hardcoded, or can we rely on the developers default e.g. if they're using an internal mirror?

No, it should not be hard-coded, I only convert the entries in the old config file (https://github.com/jupyterhub/jupyter-server-proxy/blob/main/labextension/.yarnrc). For other extensions only the `nodeLinker` config is required.